locales/en-US.json

```diff
@@ -1,500 +1,503 @@
 {
-    "Add to playlist": "Add to playlist",
-    "Add to playlist: ": "Add to playlist: ",
-    "Answer": "Answer",
-    "Search for videos": "Search for videos",
```

Comment:
Please do not change the indentation level in this file!
It will break weblate, the translation tool!